src/Uno.UI/Helpers/WinUI/CppWinRTHelpers.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Documents;
+
+namespace Uno.UI.Helpers.WinUI
+{
+	internal static class CppWinRTHelpers
+	{
+		public static bool SetFocus(DependencyObject obj, FocusState focusState)
+		{
+			if (obj != null)
+			{
+				// Use TryFocusAsync if it's available.
+				if (false) //TODO Uno specific: WinUI checks for TryFocusAsync method presence, which is not implemented yet in Uno (issue #4256)
```

Comment:
Codacy found an issue: [Remove this useless 'if' statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6063676)